tests/php/View/Parsers/ShortcodeParserTest.php

```diff
@@ -79,37 +79,54 @@ public function testNotRegisteredShortcode()
         );
     }
 
-    public function testSimpleTag()
+    public function simpleTagDataProvider()
     {
-        $tests = [
-            '[test_shortcode]',
-            '[test_shortcode ]', '[test_shortcode,]', '[test_shortcode, ]' . '[test_shortcode/]', '[test_shortcode /]', '[test_shortcode,/]', '[test_shortcode, /]'
+        return [
+            ['[test_shortcode]'],
+            ['[test_shortcode ]'],
+            ['[test_shortcode,]'],
+            ['[test_shortcode, ][test_shortcode/]'],
+            ['[test_shortcode /]'],
+            ['[test_shortcode,/]'],
+            ['[test_shortcode, /]']
         ];
+    }
 
-        foreach ($tests as $test) {
-            $this->parser->parse($test);
-
-            $this->assertEquals([], $this->arguments, $test);
-            $this->assertEquals('', $this->contents, $test);
-            $this->assertEquals('test_shortcode', $this->tagName, $test);
-        }
+    /**
+     * @dataProvider simpleTagDataProvider
+     */
+    public function testSimpleTag($test)
```

Comment:
Refactored the test to use a data provider. We're still testing exactly the same scenarios.